test/github-trigger

```diff
@@ -16,6 +16,10 @@ def main():
 
     github = testinfra.GitHub()
 
+    if opts.context not in testinfra.DEFAULT_VERIFY:
```

Comment:
Yes i thought that too. It turns out one ends up using force a lot routinely with this script, and that would break the check.